src/store/modules/layers.store.js

```diff
@@ -37,6 +37,17 @@ const state = {
      * @type AbstractLayer
      */
     previewLayer: null,
+    /**
+     * Year being picked by the time slider. The format is YYYY (but might evolve in the future in
+     * the ISO 8601 date format direction, meaning YYYY-MM-DD, hence the String type).
+     *
+     * We store it outside the time config of layers so that layers revert back to their specific
+     * chosen timestamp when the time slider is closed. That also means that the year picked by the
+     * time slider doesn't end up in the URL params too.
+     *
+     * @type Number
+     */
+    previewYear: null,
```

Comment:
This might be the cause of the bug describe with the print screen above. Why not changing the URL with the slider ?